src/mscorlib/src/System/Globalization/CultureInfo.cs

```diff
@@ -26,25 +26,29 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Contracts;
+using System.Runtime.Serialization;
+using System.Threading;
+#if FEATURE_APPX && !FEATURE_COREFX_GLOBALIZATION
```

Comment:
FEATURE_COREFX_GLOBALIZATION is mainly reflecting Linux. this is how is defined awhile ago. I agree it is not needed here. I'll remove it